runtime/lua/vim/lsp.lua

```diff
@@ -50,6 +71,16 @@ local function resolve_bufnr(bufnr)
   return bufnr
 end
 
+--@private
+--- callback called by the client when trying to call a method that's not
+--- supported in any of the servers registered for the current buffer.
+--@param method (string) name of the method
+local function unsupported_method(method)
```

Comment:
let's make this not local.

Then people can just override it.

`vim.lsp._unsupported_method` or something like that maybe? That way people can easily override the functionality (or silence the errors if they don't care)

Reply:
Wouldn't that make it part of the public API and difficult to change in the future? If we do end-up making it public, should it be part of the callbacks module? Or should we keep that module for callbacks to methods that exist in the LSP spec?